### PR TITLE
RF_01.006.05 | FreestyleProject > Move project from Dashboard to Folder

### PR DESCRIPTION
**Pull request by antonbloha**
[RF_01.006.05](https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/534)

### CODE REVIEW

File: cypress/e2e/freestyleProjectMoveProjectPOM.cy.js [hunk 1]
```diff
@@ -99,4 +125,30 @@ describe('US_01.006 | FreestyleProject > Move project', () => {
      
     });
 
-});
\ No newline at end of file
+    it('RF_01.006.05 | Move project from the Dashboard to Folder', () => {
```
Comment (Stavolg) [inline cypress/e2e/freestyleProjectMoveProjectPOM.cy.js:128]: what is the difference between this TC and rows 54-79? why do you write it twice?

File: cypress/pageObjects/DashboardPage.js [hunk 2]
```diff
@@ -137,6 +140,10 @@ class DashboardPage {
     return this
   }
 
+  clickMoveTheProjectButton() {
+    this.getMoveTheProject().click()
+    return FreestyleProjectPage
```
Comment (Stavolg) [inline cypress/pageObjects/DashboardPage.js:145]: we decided to use "return this" only

**Review by irinadash — APPROVED** [review]

File: cypress/e2e/freestyleProjectMoveProjectPOM.cy.js [hunk 3]
```diff
@@ -51,6 +51,32 @@ describe('US_01.006 | FreestyleProject > Move project', () => {
         freestyleProjectPage.getProjectInfoSection().should('contain', `Full project name: ${selectedFolder}/${newJobPageData.projectName}`)
     });
 
+    it('TC_01.006.05 | Move project from Dashboard to Folder', () => {
+        dashboardPage.clickNewItemMenuLink();
+        newJobPage.typeNewItemName(newJobPageData.projectName)
```
Comment (Stavolg) [inline cypress/e2e/freestyleProjectMoveProjectPOM.cy.js]: newJobPageData replace to project.name and/or folder.name

File: cypress/pageObjects/FolderPage.js [hunk 4]
```diff
@@ -10,14 +10,15 @@ class FolderPage extends Header {
     getFolderNameOnMainPanel = () => cy.get("#main-panel h1");
     getDashboardBreadcrumbsLink = () => cy.get('#breadcrumbs a[href="/"]');
     getNewItemMenuOption = () => cy.get('[href $= "/newJob"]');
+    getProjectName = () => cy.get('.jenkins-table__link > span');
```
Comment (Stavolg) [inline cypress/pageObjects/FolderPage.js:13]: pls connect to author row 15 and then do refactor

File: cypress/e2e/freestyleProjectMoveProjectPOM.cy.js (hunk 1 above)
Comment (antonbloha) [reply cypress/e2e/freestyleProjectMoveProjectPOM.cy.js]: Fixed.

File: cypress/pageObjects/DashboardPage.js (hunk 2 above)
Comment (antonbloha) [reply cypress/pageObjects/DashboardPage.js]: Fixed.

File: cypress/e2e/freestyleProjectMoveProjectPOM.cy.js (hunk 3 above)
Comment (antonbloha) [reply cypress/e2e/freestyleProjectMoveProjectPOM.cy.js]: Fixed.

File: cypress/pageObjects/FolderPage.js (hunk 4 above)
Comment (antonbloha) [reply cypress/pageObjects/FolderPage.js]: Fixed.

**Review by Stavolg — APPROVED** [review]

**Review by irinadash — APPROVED** [review]
Looks good

**Pull request merged**